packages/fluentui/react-northstar/src/components/Toolbar/ToolbarMenuItem.tsx

```diff
@@ -336,7 +337,9 @@ export const ToolbarMenuItem = compose<'button', ToolbarMenuItemProps, ToolbarMe
         ) : (
           <>
             {createShorthand(composeOptions.slots.icon, icon, { defaultProps: () => slotProps.icon })}
-            {content}
+            {createShorthand(composeOptions.slots.content, content, {
+              defaultProps: () => getA11yProps('content', slotProps.content),
```

Comment:
Do we really need `getA11yProps` here? And if so is there any `content` slot in the behavior?

Reply:
Hi, no there's no content slot in behavior. 
I did it like this because `MenuItem` wraps `getA11yProps` around its content too, without having a content slot in behavior. `ToolbarMenuItem` is using `menuItemBehavior`. So if one day a behavior is added to menuItem content (althought I doubt), `ToolbarMenuItem` content could have as well.

Reply:
I get that, just wondering if it's really worth keeping it then. Like if I am reading it first time I would have to go to behaviors to check what will happen and then find out that nothing actually happens. So maybe for readability better to just keep simple `defaultProps: () => slotProps.content` ? WDYT @ling1726 @layershifter ?

Reply:
I have no strong opinions here and it's not a big issue for me. If I were to be doing the implementation I would no include it if it were useless.

Sorry for being so indecisive 😅

Reply:
I just had a call with shift, and he has a good point that user could apply their own a11y behavior. And in such case, wrapping content with getA11yProps is needed for the customized behavior to work.
(I'm so happy I'm doing something correct without knowing it :D )